### ISSUE TITLE
composer.json requires incorrect zendframework/zend-stdlib

### ISSUE THREAD

**Issue by NotionCommotion** [concrete5/composer#30]
Hi,, I just did a brand new install of concrete5 8.5.1 on PHP 7.3.6 using Composer 1.8.5 per https://documentation.concrete5.org/developers/installation/installation.

```
composer create-project -n concrete5/composer hello_world
./vendor/bin/concrete5 c5:install -i
```

Front end is fine, but when logging on to backend using http://example.com/index.php/dashboard/welcome, get the following.

```
Whoops\Exception\ErrorException thrown with message ""continue" targeting switch is equivalent to "break". Did you mean to use "continue 2"?"
Stacktrace:
#39 Whoops\Exception\ErrorException in /var/www/hello_world/vendor/zendframework/zend-stdlib/src/ArrayObject.php:426
...
```

This issue was fixed by https://github.com/zendframework/zend-stdlib/commit/d0cd1d043ba70feb503fb1015fbc94a6ca414056, (3.2.1) however, Composer will only install version 2.7 which is its own branch release.  Note sure the best course of action as 3.2.1 is a pre-release.

Thanks!

```
https://packagist.org/packages/concrete5/composer
https://github.com/concrete5/composer
composer/composer.json
  "require": {
    ...
    "concrete5/core": "^8.2",
    ...
  },
https://packagist.org/packages/concrete5/core
https://github.com/concrete5/concrete5-core
concrete5-core/composer.json
  "require": {
    ...
	"concrete5/zend-queue": "0.9.*",
    ...
  },
 https://packagist.org/packages/concrete5/zend-queue
 https://github.com/concrete5/ZendQueue
    "require": {
        "php": ">=5.3.3",
        "zendframework/zend-stdlib": "~2"
    },
https://packagist.org/packages/zendframework/zend-stdlib
https://github.com/zendframework/zend-stdlib
=> Fixes the issue: using break instead of continue 2 inside switch, master (#91)  release-3.2.1, Commits on Jul 10, 2018
```

**Comment by mlocati** [comment]
You should add (and enable) the concrete5 dependency patches package.
See https://github.com/concrete5/concrete5/issues/7934#issuecomment-506593513

**Comment by NotionCommotion** [comment]
Thank you mlocati!

All working.  Just to be sure, I should add these two items in https://github.com/concrete5/composer/blob/master/composer.json which was downloaded to my root directory when executing `composer create-project -n concrete5/composer hello_world`, and not https://github.com/concrete5/concrete5/blob/develop/composer.json, correct?

It would be nice if either the first composer.json file had this when downloaded or it was at least documented at github/packagist.

Regardless, all good, and appreciate the help!

**Comment by mlocati** [comment]
Korvin just merged the required changes: https://github.com/concrete5/composer/pull/29

**Comment by NotionCommotion** [comment]
Thank you

Closes #30
